discovery/azure/azure.go

```diff
@@ -157,29 +159,48 @@ type azureClient struct {
 	vmssvm compute.VirtualMachineScaleSetVMsClient
 }
 
+// getAzureEnvironment returns an instance of azure.Environment based on envName.
+func getAzureEnvironment(envName string) (azure.Environment, error) {
+	env, err := azure.EnvironmentFromName(envName)
```

Comment:
The next 4 lines are the same as `return azure.EnvironmentFromName(envName)`. In the end I don't think we need this function, just use `azure.EnvironmentFromName(cfg.Environment)` below in `createAzureClient()`.

Reply:
Ah yes. Done